default.xml

```diff
@@ -25,4 +25,5 @@
   <project name="openembedded/meta-openembedded" path="layers/meta-openembedded" remote="github"/>
   <project name="openembedded/openembedded-core" path="layers/openembedded-core" remote="github"/>
   <project name="git/meta-security" path="layers/meta-security" remote="yocto"/>
+  <project name="sbabic/meta-swupdate" path="layers/meta-swupdate" remote="github" revision="warrior"/>
```

Comment:
We should rely on the default `revision` setting so that we won't have to change this line when we move away from warrior.
```suggestion
  <project name="sbabic/meta-swupdate" path="layers/meta-swupdate" remote="github"/>
```